src/main/scala/latis/service/hapi/Latis3Interpreter.scala

```diff
@@ -169,9 +183,10 @@ class Latis3Interpreter extends HapiInterpreter[IO] {
   private def makeProjection(params: Option[NonEmptyList[String]]): List[UnaryOperation] =
     params.map { ps =>
       val names: List[String] = "time" :: ps.toList
-      List(Projection(names:_*))
+      List(Projection(names.mkString(",")))
```

Comment:
Might as well just use the `Projection.fromArgs` constructor since we already have a list?

Reply:
I considered that. I couldn't decide if `.mkString(",")` was worse than `.fold(throw _, identity)` or not. 
Changed it to
```scala
List(Projection.fromArgs(names).fold(throw _, identity))
```

Reply:
Oh, I didn't notice that `fromArgs` could fail.

I guess what we should do is have a ticket for the future to use `Identifier` everywhere. Then we can get rid of all the places we have to throw or return an empty string.